src/Compilers/CSharp/Test/Symbol/Symbols/SymbolEqualityTests.cs

```diff
@@ -41,6 +42,11 @@ void M()
             var nonNullPlus = (IMethodSymbol)model.GetSymbolInfo(invocations[0]).Symbol;
             var nullPlus = (IMethodSymbol)model.GetSymbolInfo(invocations[1]).Symbol;
 
+            Assert.Equal(MethodImplAttributes.IL, nonNullPlus.ImplementationAttributes);
+            Assert.Equal(MethodImplAttributes.Managed, nonNullPlus.ImplementationAttributes);
```

Comment:
Since `Managed` and `IL` are the same value, we don't need to compare to both. Just one is fine.